app/components/reasons_for_rejection_component.html.erb

```diff
@@ -30,9 +30,10 @@
           <li>
             <% if reason == 'other' %>
               <p><%= reasons_for_rejection.quality_of_application_other_details %></p>
-              <p><%= reasons_for_rejection.quality_of_application_other_what_to_improve %></p>
+              <p>Details: <%= reasons_for_rejection.quality_of_application_other_what_to_improve %></p>
```

Comment:
These `Details` prefixes don't appear on the Provider design - is this intended to change in both views? I know we iterated a similar `Advice:` prefix out of the provider designs...

Reply:
Yes it does update the provider design. I've done it that way mainly because the provider interface has words to the effect _this is what the candidate sees_ (maybe a bad assumption). But those _Details: _ prefixes can be removed when rendered in the provider interface if @adamsilver or others think they should be?

Reply:
Yes my assumption was that apart from the `Change` links this should be the same rendering in Candidate and Provider UIs.

Reply:
Yep, candidates and providers should see the same content.

The use of ‘Details: ’ prefix also appears to be used inconsistently; sometimes it appears for a free text answer, sometimes it doesn’t. Is that by design @adamsilver? I’d be tempted to remove it.

Reply:
@johndoates didn't we want to remove these prefixes?

Reply:
Yes we want to remove the prefixes. They don't add anything, given that there's a heading to give context. It's reasonable to assume that under a heading such as 'Something you did' you'll find details of what you did.

Reply:
`Words:`